core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -1172,18 +1177,21 @@ private function encodeHtmlAttributes($html)
 
 			$tag = $matches[0][0];
 
-			// Encode insert tags
-			$tagPrefix = substr($tag, 0, $matches[1][1] - $matches[0][1] + \strlen($matches[1][0]));
-			$tag = $tagPrefix . $this->fixUnclosedTagsAndUrlAttributes(substr($tag, \strlen($tagPrefix)));
-			$tag = preg_replace('/(?:\|attr)?}}/', '|attr}}', $tag);
-			$tag = str_replace('|urlattr|attr}}', '|urlattr}}', $tag);
+			if (strpos($tag, '{{') !== false || strpos($tag, '}}') !== false)
```

Comment:
Just a random thought: This would search the same string twice. Maybe a regex with `{{|}}` is faster?

Reply:
Ah, it's just a very short string, then probably not :D

Reply:
> Ah, it's just a very short string

Yes, it only checks one matched tag like `<span title="{{foo}}" class="bar">`